akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -505,7 +514,7 @@ private[stream] final class GraphInterpreterShell(
       _isTerminated = true
```

Comment:
Having the comment on the actual catch-and-ignore would be highly advisable, don't you agree?